fastlane/Fastfile

```diff
@@ -15,14 +15,14 @@ derived_data_path = 'derived_data'
 source_packages_path = 'spm_cache'
 buildcache_xcargs = 'CC=clang CPLUSPLUS=clang++ LD=clang LDPLUSPLUS=clang++'
 is_localhost = !is_ci
-@force_check = false
+@force_check = true # FIXME
```

Comment:
Why FIXME? 🤔

Reply:
To set it back to `false` after testing 🤠